### PR TITLE
Add: Interactions for like button

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
Display the number of likes for each item on the homepage.

### CODE REVIEW

**Review by ShoiraTa — COMMENTED** [review]
Hi @Danieldotcomcoder ,

:thumbsup: This is good to make it this far
:thumbsup: The app feature is working as expected.

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes  :recycle: 
_Check the comments under the review._ 

## Optional suggestions
- [ ] it is not recommended to start functions from capital letters, because capitalization is generally applied to identify classes

File: dist/d25b18980b38734775d9.svg [hunk 1]
```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (ShoiraTa) [inline dist/d25b18980b38734775d9.svg:1]: - [ ] It is recommended to keep project clean by deleting not used assets. please kindly delete this svg file to keep our project clean

Comment (Danieldotcomcoder) [reply dist/d25b18980b38734775d9.svg]: Thank you, I did fix this issue 👍

**Comment by Danieldotcomcoder** [comment]
Thank you for your comments, I did fix the suggested changes 😀

**Comment by ShoiraTa** [comment]
👍🏻 👍🏻 👍🏻  time to merge!

**Pull request merged**